README.md

```diff
@@ -78,6 +113,35 @@ Steps:
         cd easy-stage-dataset
         mvn install
 
+TEST NOTES
+----------
+
+No tests are available that mock fedora or the database for files and folders.
+So run integration tests to check for regression.
+
+Run `./stageFileItem.sh out/file-sdos src/test/resources/example.csv`
+to cover the logic that checks existence of datasets and folders in the repository or folder-SDO's on the file system.
+Reading a CSV is also covered with a unit-test, creation of file/folder SDO's is covered by stageDataset.
+
+
+Steps for a regression test:
+
+* Create `apphome.sh` in the project root to override the var `APPHOME`
+  or create a `home` folder in the root of the project.
+  See also [installation and configuration](#installation-and-configuration).
+* Remove the content of the directory `out` (ignored by git like the above)
+* compile the old code with maven
+* Process any example-bag you may find, for example:
+  * `./stageDataset.sh -t2015 -uURN -dDOI src/test/resources/example-bag out/local-sdo`
+  * `./stageDataset.sh -t2015 -uURN -dDOI ../easy-deposit/src/test/resources/simple/example-bag out/simple-sdo`
+* Move the created `out` to another location, say `~/old`.
+* Repeat until OK:
+  * compile the new code with maven
+  * Clear the folders in `out`.
+  * Process the same bags.
+  * Compare the results: `diff -r ~/old target/test-out`
+
+
```

Comment:
OK, now I get it. This can certainly be a useful trick when testing. Perhaps it is more suitable for a wiki page. It is up to the tester to find out whether any differences in the outputs are intended or not, of course.